src/bin/tectonic.rs

```diff
@@ -19,49 +20,91 @@ use tectonic::status::{ChatterLevel, StatusBackend};
 
 use tectonic::{ctry, errmsg, tt_error, tt_error_styled, tt_note};
 
-fn inner(
-    args: ArgMatches,
-    config: PersistentConfig,
-    status: &mut TermcolorStatusBackend,
-) -> Result<()> {
+#[derive(Debug, StructOpt)]
+#[structopt(name = "Tectonic", about = "Process a (La)TeX document")]
+struct Opt {
```

Comment:
It looks like this is the structure named recommended by structopt, but I'm not a big fan. How about `CliOptions`?

Reply:
Yes, quite the better name!